### PR TITLE
[NEW] l10n_br_portal base

### PR DESCRIPTION
**Pull request by mileo**

### CODE REVIEW

**Review by marcelsavegnago — APPROVED** [review]

**Comment by mileo** [comment]
@marcelsavegnago obrigado, estou tentando arrumar os testes para manter o coverage. 

Mas esta foda, pois não consigo rodar na minha maquina o teste do JS....

**Review by sadamo — APPROVED** [review]

**Review by rvalyi — APPROVED** [review]
Looks all great! thanks for this new module!

File: l10n_br_portal/README.rst [hunk 1]
```diff
@@ -0,0 +1,91 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
```
Comment (renatonlima) [inline l10n_br_portal/README.rst]: Please, create readme folder and its files to auto generate this file.

File: l10n_br_portal/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    'depends': [
+        'portal',
+        'l10n_br_zip',
+    ],
+    'data': [
+        'views/assets.xml',
+
+        'views/portal_templates.xml',
+    ],
+    'demo': [
```
Comment (renatonlima) [inline l10n_br_portal/__manifest__.py]: You can remove empty keys in manifest.

File: l10n_br_portal/README.rst [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
```
Comment (renatonlima) [inline l10n_br_portal/README.rst]: Please, create readme folder and its files to auto generate this file.

File: l10n_br_portal/__manifest__.py (hunk 2 above)
Comment (renatonlima) [inline l10n_br_portal/__manifest__.py]: You can remove empty keys from module manifest.

**Review by renatonlima — CHANGES_REQUESTED** [review]

**Review by renatonlima — APPROVED** [review]

**Pull request merged**